azure-templates/deploy-steps.yml

```diff
@@ -37,6 +37,11 @@ parameters:
     type: string
     default: '0.5'
 
+  # the resource group must be the same where vnet is created
+  - name: 'containerInstanceResourceGroup'
+    type: string
+    default: 'io-p-rg-common'
```

Comment:
Can't we get this value from the environment?

Reply:
`az network vnet subnet list` requires resource group to list subnets, so we can't get values from other resources.

I don't think that we'll changes these values often.

```
  # the resource group must be the same where vnet is created
  - name: 'containerInstanceResourceGroup'
    type: string
    default: 'io-p-rg-common'
  # attached vnet to the container instance
  - name: 'containerInstanceVNet'
    type: string
    default: 'io-p-vnet-common'
  # container instance subnet
  - name: 'containerInstanceSubnet'
    type: string
    default: 'azure-devops'
```